sros2/test/sros2/commands/security/verbs/test_generate_policy_no_nodes.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+
+from ros2cli import cli
+
+
+def test_generate_policy_no_nodes(capsys):
```

Comment:
I'm not sure that this PR resolves the previous test failure on Windows (https://github.com/ros2/sros2/issues/143).
We probably still want to keep the TODO and skip decorator. Note, to reproduce the failure, you should also run tests for packages other than sros2 (e.g. `colcon test --packages-select demo_nodes_cpp && colcon test --packages-select sros2`).